x-pack/platform/packages/shared/kbn-ai-assistant/src/conversation/conversation_view.tsx

```diff
@@ -24,6 +25,7 @@ import { useConversationList } from '../hooks/use_conversation_list';
 
 const SECOND_SLOT_CONTAINER_WIDTH = 400;
 
+const queryClient = new QueryClient();
```

Comment:
Is there a better place to do this, instead of asking every consumer of the flyout to add a query client?

Reply:
I ended up moving the QueryClientProvider to `getAddConnectorFlyoutLazy`  instead  [here](https://github.com/elastic/kibana/pull/217740/commits/e8736785dd29a408017258576ed5bb40f309c4e7) which should be a better spot, and more future proof.